cmd/gokeyless-cli-tester/gokeyless-cli-tester.go

```diff
@@ -50,6 +60,55 @@ func main() {
 	if err != nil {
 		log.Fatal("failed to load cert directory:", err)
 	}
+	if delegate {
```

Comment:
This is basically a standalone function, but you've wrapped it in an if statement. I'd move this code into its own function, and move the c.ScanDir call closer to `for _, priv := range privs` than the invocation of this.